bedrock/firefox/templates/firefox/product-base.html

```diff
@@ -8,16 +8,6 @@
   {% stylesheet 'firefox-product-page' %}
 {% endblock %}
 
-{% block site_footer %}
-  {% with footer_class='dark' %}
-    {% if LANG.startswith('en-') %}
-      {% include 'includes/site-footer-new.html' %}
-    {% else %}
-      {% include 'includes/site-footer.html' %}
-    {% endif %}
-  {% endwith %}
-{% endblock %}
-
```

Comment:
Hmm, you're quite right. I wonder if this got orphaned during a rebase? it certainly seems unused, so I'll delete it 👍